toolchains/cc/cc.nix

```diff
@@ -298,6 +298,7 @@ pkgs.runCommand "bazel-${cc.orignalName or cc.name}-toolchain"
         if [[ -x ${cc}/bin/ld.gold ]]; then echo True; else echo False; fi
       )
     )
+    COVERAGE_LINK_FLAGS=()
```

Comment:
why is this needed? It overrides the [set of flags defined above](https://github.com/tweag/rules_nixpkgs/pull/575/files#diff-ebda446a0ba42acd64ea7fe331d17801b2c3f88c96e3fe1d785f95345404523dR288).

Reply:
```suggestion
    EXTRA_FLAGS_PER_FEATURE=()
```